src/components/WeightedResultBar/__tests__/WeightedResultBar.spec.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import { WeightedResultBar } from '../WeightedResultBar';
+
+describe('WeightedResultBar', () => {
+    it('should render correctly in loading state', () => {
+        const wrapper = mount(<WeightedResultBar>Result</WeightedResultBar>);
```

Comment:
The unit title is misleading here. Also there are no values passed to the component which breaks calculations in the snapshot below.

Reply:
uupps 👍